### DISCUSSION TITLE
Script closes after extracting dependencies

### DISCUSSION THREAD

**Discussion by Marrus-Source** [illsk1lls/ZipRipper#39] [Q&A]
OpenCL enabled.  No error, no indication the program ran or crashed, it just closes.  Same with either online or offline mode.

Are there any logs generated that might help with finding out why? I don't see any being generated.

EDIT: After some testing, I've determined it has something to do with the .zip files themselves.  Made a new compressed file with password, it worked fine.  Tried a locked .rar file, that worked too.  Are there limitations for file types or encryption methods not mentioned?

**Comment by illsk1lls** [comment]
Are there any special characters in the ZIP filename? Can you see the filenames inside or are they hidden behind the password?

It could also be the archive, its common for some other compression tools to use the ZIP extension when they are not in fact zip files..  But it should tell you it is not supported instead of crashing, so I'm guessing maybe special chars in the archive?  I need more details

**Reply by Marrus-Source** [reply]
I'm guessing this is the problem.  Both the file name and the files inside have a • character in the name.

I don't see any mention of characters that aren't allowed in the program description, so I didn't think naming conventions were an issue.

**Reply by illsk1lls** [reply]
i might be able to get it working, it depends if its my script or John that is having the issue, it is probably me I will make a test archive and see what I can come up with
